protocol/scripts/genesis/prod_pregenesis.sh

```diff
@@ -19,34 +19,34 @@ fi
 # Capture the required argument
 DYDX_BINARY="$1"
 
+source "./testing/genesis.sh"
+
 TMP_CHAIN_DIR="/tmp/prod-chain"
 TMP_EXCHANGE_CONFIG_JSON_DIR="/tmp/prod-exchange_config"
 NINE_ZEROS="000000000"
 EIGHTEEN_ZEROS="$NINE_ZEROS$NINE_ZEROS"
 BRIDGE_MODACC_BALANCE="1$NINE_ZEROS$EIGHTEEN_ZEROS" # 1e27
 BRIDGE_MODACC_ADDR="dydx1zlefkpe3g0vvm9a4h0jf9000lmqutlh9jwjnsv"
 
-source "./testing/genesis.sh"
-
 # TODO(GENESIS): Update below values before running this script. Sample values are shown.
 ################## Start of required values to be updated ##################
-CHAIN_ID="dydx-1"
+CHAIN_ID="dydx-sample-1"
 # Base denomination of the native token. Usually comes with a prefix "u-", "a-" to indicate unit.
-NATIVE_TOKEN="adv4tnt"
+NATIVE_TOKEN="asample"
 # Denomination of the native token in whole coins.
-NATIVE_TOKEN_WHOLE_COIN="dv4tnt"
+NATIVE_TOKEN_WHOLE_COIN="asample"
```

Comment:
Should this be the same as `NATIVE_TOKEN` or should this be `sample`?

Reply:
Nice catch

Reply:
Updated